polaris/polaris/deposit/views.py

```diff
@@ -83,18 +86,15 @@ def _verify_optional_args(request):
 @xframe_options_exempt
 @api_view(["GET", "POST"])
 @renderer_classes([TemplateHTMLRenderer])
+@validate_interactive_jwt()
```

Comment:
It seems like someone could validate their own account, but then request info on a Transaction that they don't own.  As long as there is a session authenticated, they could request any transaction ID in the function below.

Reply:
Thats true. One idea is to not only save `request.session["authenticated"]` but also `request.session["account"]`. Then in `POST` requests, check that the requested transaction belongs to the account on the request.

That seems to solve this issue, but let me know if you have a better solution.

Reply:
yeah i think that's exactly right, session should store some user identifier